src/libgrate/fragment_asm_parser.y

```diff
@@ -765,6 +767,19 @@ TEX_INSTRUCTION:
 	|
 	;
 
+TEX_LOD_REG:
+	',' T_ROW_REGISTER
+	{
+		asm_tex_instructions[asm_fs_instructions_nb].enable_bias = 1;
+		yyval.u = $2;
+	}
+	|
+	{
+		asm_tex_instructions[asm_fs_instructions_nb].enable_bias = 0;
+		yyval.u = 0xDEAD;
```

Comment:
The "common" way of doing this, is to consider these two different opcodes, [TEX](http://gallium.readthedocs.io/en/latest/tgsi.html#opcode-TEX) and [TXB](http://gallium.readthedocs.io/en/latest/tgsi.html#opcode-TXB)...